citar.el

```diff
@@ -165,6 +171,34 @@ and nil means no action."
   :group 'citar
   :type 'function)
 
+(defcustom citar-major-mode-functions
+  '(((latex-mode) .
+     ((local-bib-files . citar-latex--local-bib-files)
+      (keys-at-point . citar-latex--keys-at-point)))
+    ((org-mode) .
+     ((local-bib-files . org-cite-list-bibliography-files)
+      (insert-keys . citar--insert-keys-org-cite)
+      (keys-at-point . citar-get-key-org-cite))))
+  "The variable determining the major mode specifc functionality.
+It is alist with keys being a list of major modes. The value is an alist
+with values being functions to be used for these modes while the keys
+are symbols used to lookup them up. The keys are
+
+local-bib-files: the corresponding functions should return the list of
+local bibliography files.
```

Comment:
This cannot now be nil. What if a mode (like markdown) doesn't support local bibs?